### DISCUSSION TITLE
How can I change the port number ? [Jul 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6129] [Discord Threads]
> 🙋‍♂️ **r4nchy**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### You can define `PORT` environment variable  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1122760317865300050)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
